configs/zhinst.json

```diff
@@ -23,13 +43,40 @@
         "selector": "//nav[@class='nav-menu']/h3/a",
         "type": "xpath",
         "global": true,
-        "default_value": "Zurich Instrument"
+        "default_value": "Zurich Instruments"
       },
       "lvl1": ".doc h1",
       "lvl2": ".doc h2",
       "lvl3": ".doc h3",
       "lvl4": ".doc h4",
       "text": ".doc p, .doc td.content, .doc th.tableblock"
+    },
+    "api": {
+      "lvl0": {
+        "selector": "//li[@class='toctree-l1 current active nav-item']/a",
```

Comment:
Hey @tobiasah, we plan to deprecate the xPath support and provide an access to a Cheerio instance in the next major update. Do you think you could find a DOM alternative in the meantime? This way, your config will still be close to correct once it has been migrated

Reply:
Thanks for the input.
I wasn't aware that path will be removed I just used it because I find it a bit more clear. 
I switched the xpath selectors to DOM.

Reply:
Thanks! We will provide some more informations soon!